lib/icinga2/downtimes.rb

```diff
@@ -113,6 +113,46 @@ def add_downtime( params )
       )
     end
 
+    # remove downtime
+    #
+    # @param [Hash] params
+    # @option params [String] downtime_name
+    #
+    # @example
+    #    param = {
+    #      downtime_name: 'test',
+    #    }
+    #    remove_downtime(param)
+    #
+    # @return [Hash]
+    #
+    def remove_downtime( params )
```

Comment:
I haven't documented it yet.
for individual parameters please do not pass hashes:

eg.
(wrong) remove_downtime(params) 
(okay) remove_downtime(downtime_name)